controllers/challengeController.js

```diff
@@ -38,24 +39,26 @@ const sendChallengeResponse = async (req, res) => {
   }
   return ''
```

Comment:
Unreachable return, if needed we can return from the `finally` block.

Reply:
Is it even needed?
The `catch` block will take care, right?